### DISCUSSION TITLE
Lodash update

### DISCUSSION THREAD

**Discussion by marek-tabor** [jaredpalmer/formik#2937] [Q&A]
Hi, 

I would like to ask if you can Upgrade lodash to version 4.17.20 or higher? Formik uses an old version of lodash which according to Snyk is not very secure https://snyk.io/vuln/SNYK-JS-LODASH-590103.

If that is a big issue for you, is there a way to force point formik to use lodash version provided by myself?

**Comment by TrySound** [comment]
Formik does not restrict lodash to specific version. You can uninstall and install formik again to update lockfile.

**Reply by marek-tabor** [reply]
Glad to hear that! thank you

**Comment by maddhruv** [comment]
Hey @marek-tabor thanks for raising this concern, we have raised a PR bumping lodash to the latest - https://github.com/formium/formik/pull/2938
Should be released in an upcoming version 👍 

Feel free to mark the discussion as resolved, when you are able to use the latest lodash

**Reply by marek-tabor** [reply]
Thanks! will do, after the new version is released :)
